### DISCUSSION TITLE
Backend API querying/sorting not working

### DISCUSSION THREAD

**Discussion by jordanmcgregor** [swellstores/community#453] [Help] [answered]
I am modeling the example from the [backend documentation](https://developers.swell.is/backend-api/querying/sorting#logical-operators) almost verbatim and it isn't working. Here is the example from the backend docs:

await swell.get('/orders', {
  where: {
    date_created: { $gt: '2022-07-01T00:00:00Z' },
    date_created: { $lt: '2023-06-30T00:00:00Z' }
  }
});

And here is the code that I have written. 

const orders = await swell.get('/orders', {
    where: {
      date_created: { $gte: '2023-12-02T00:29:31.383Z' },
      account_id: account_id,
    },
    limit: 25,
    page: 1
  });

I also tried the following permutations without any luck:

const orders = await swell.get('/orders', {
    where: {
      date_created: { $gte: '2023-12-02T00:29:31.383Z' },
      account_id: account_id,
    },
  });

const orders = await swell.get('/orders', {
    where: {
      date_created: { $gte: '2023-12-02T00:29:31.383Z' },
    },
  });

const orders = await swell.get('/orders', {
    where: {
      date_created: { $gt: '2023-12-02T00:29:31.383Z' },
    },
  });

Each permutation return all orders. What am I missing here.

**Comment by sags95** [comment, marked as answer]
I'm not able to reproduce an issue here with filtering, are you able to filter correctly when using the API console in the dashboard?

**Reply by jordanmcgregor** [reply]
I am able to successfully filter using the API console in the dashboard. I just am not able to successfully filter using the backend api from my project.

**Reply by ericingram** [reply]
Could you share the exact code you're testing with?

**Reply by logeshswell** [reply]
Also, please share the swell-node version that you use.
